apps/docs/.vitepress/config.ts

```diff
@@ -164,6 +164,10 @@ export default defineConfig({
                 text: 'Date Time',
                 link: '/guide/types/date-time',
               },
+              {
```

Comment:
Is there an order to this list?
Could this be higher? Seems like valuable knowledge.